### DISCUSSION TITLE
[Feature Request] Input - error check

### DISCUSSION THREAD

**Discussion by serloi** [element-plus/element-plus#1183] [Ideas]
Hello there!

I'm actually switching vue2/Bootstrap vue to vue3/..?.. from scratch for new projects  and I'm testing lots of UI libraries ready with vue3 and using slots. Element-plus seems very cool and is beautiful!

But a feature is missing for input (or other components why not), the possibility to check the value and color green or red then component if ok or not ok when user type text.

Have a good day!

Serge

**Comment by zazzaz** [comment]
Thanks for using Element UI. Feel free to ask us for any help if needed.

For this feature request, and live demo or similar component for us to check?

Or maybe a form validation will suit your need? https://element-plus.org/#/en-US/component/form#validation

**Reply by serloi** [reply]
Oups, I didn't see this part (only check "Input" in Form section part)
Well so it's ok ! :)
Thank you for your very fast response 👍
